source/fundamentals/connection/tls.txt

```diff
@@ -82,12 +82,20 @@ you should use the JVM Trust Store and the JVM Key Store.
 Configure the JVM Trust Store
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. note::
+
+   By default, the JRE includes many commonly used public certificates
+   from signing authorities like `Let's Encrypt
+   <https://letsencrypt.org/>`__. As a result, you can connect to
+   instances of :atlas:`MongoDB Atlas <>` with TLS/SSL without configuring
+   the trust store.
+
 The JVM trust store saves certificates that securely identify other
 applications with which your Java application interacts. Using these
 certificates, your application can prove that the connection to another
 application is genuine and secure from tampering by third parties.
 
-An application that serves TLS/SSL requests needs to configure two
+An application that initiates TLS/SSL requests needs to configure two
```

Comment:
This seems to contradict the note above.  Can you say something to indicate this is necessary only when connecting to a server whose certificate is signed by an authority that is not present in the JRE's default certificate store?

Reply:
Added to both the note and the this section of the text.